src/store/reducers/appReducer.js

```diff
@@ -226,6 +272,7 @@ export const reducer = { app: redux.getReducer(initialState) };
 export const getAppReducer = state => state.app;
 export const getVersion = state => getAppReducer(state).version;
 export const getTheme = state => getAppReducer(state).theme;
+export const getDiagnosticsOptOut = state => getAppReducer(state).diagnosticsOptOut;
```

Comment:
Add a getDiagnosticsEnabled selector as well.  Then use it to avoid the double negative elsewhere in code.
```js
export const getDiagnosticsEnabled = state => !getAppReducer(state).diagnosticsOptOut;
```


Then the opt out version can be used when you are specifically dealing with showing/updating the setting.

Reply:
no double negatives now.